lib/serializables/log-event.js

```diff
@@ -32,7 +32,18 @@ function LogEventFactory (
     pretty,
     console
 ) {
+    //default turn off the colorful output to achieve tidy and clean log file.
+    //however user can still enable it via command line argument.
+    var colorEnable = false;
+    var argv = process.argv;
```

Comment:
yeah, something like that would work. Start with a default of off in this module (like you have) and enable a method to toggle it on/off. That can then be called within index.js to enable/disable as needed, and would set up a good pattern if we want/need to add additional command line options.

Be aware that nconf(https://www.npmjs.com/package/nconf) in the configuration service is *also* inspecting the process, and will set configuration values based on what it finds in those parameters. We'll want to at least be aware of potential overlapping impact there: https://github.com/RackHD/on-core/blob/master/lib/services/configuration.js#L39

Yes, I agree a command-line mechanism common to all the applications would be appropriate, so something in on-core would work. In the past we've used the library https://github.com/tj/commander.js/ for this, which proved pretty useful. It would also mean changing up all of the application index.js to do the argument processing for each application first, then driving the relevant application functionality (which is usually starting a listening service)